### PR TITLE
Sanitise sensitive job application data when syncing staging db

### PR DESCRIPTION
**Pull request by david-mears-dfe**
Staging database is available to more / different people than prod, so we need to sanitise the data by hand - i.e. hand-sanitize it.

### CODE REVIEW

File: db/scripts/sanitise.sql [hunk 1]
```diff
@@ -2,12 +2,67 @@ TRUNCATE TABLE alert_runs;
...
+
+UPDATE equal_opportunities_reports
+       SET disability_no=0,
+           disability_prefer_not_to_say=0,
+           disability_yes=0,
+           gender_man=0,
+           gender_other=0,
+           gender_prefer_not_to_say=0,
+           gender_woman=0,
+           gender_other_descriptions=NULL,
```
Comment (cpjmcquillan) [inline db/scripts/sanitise.sql:18]: Can these be set to empty arrays?

**Review by cpjmcquillan — APPROVED** [review]
Just a small comment

File: db/scripts/sanitise.sql [hunk 2]
```diff
@@ -2,12 +2,67 @@ TRUNCATE TABLE alert_runs;
 TRUNCATE TABLE emergency_login_keys;
 TRUNCATE TABLE sessions;
 
+UPDATE employments
+       SET organisation='Example Organisation',
+           job_title='Example Job Title',
+           main_duties='example@example.org';
```
Comment (cpjmcquillan) [inline db/scripts/sanitise.sql:8]: Is this one supposed to be set to an email?

File: db/scripts/sanitise.sql (hunk 1 above)
Comment (david-mears-dfe) [reply db/scripts/sanitise.sql:18]: Good shout

File: db/scripts/sanitise.sql (hunk 2 above)
Comment (david-mears-dfe) [reply db/scripts/sanitise.sql:8]: No!

**Comment by david-mears-dfe** [comment]
(The above approval dates to before significant code changes)

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**